hy/lex/parser.py

```diff
@@ -252,6 +252,14 @@ def t_identifier(p):
     except ValueError:
         pass
 
+    if '/' in obj:
+        try:
+            lhs, rhs = obj.split('/')
+            return HyExpression([HySymbol('fraction'), HyInteger(lhs),
+                                 HyInteger(rhs)])
+        except:
```

Comment:
`except ValueError:`

(In accordance with the platitude that it's usually better to catch only the specific exceptions the programmer actually expects to happen, giving us the opportunity to notice and correct the erroneous program if something else happens,  and a bare `except:` will swallow even `SystemExit` and `KeyboardInterrupt`. Here, we get a `ValueError` both when `obj` has a non-one number of slashes, and when `lhs` or `lrs` can't be interpreted as an integer; I can't think of any other exceptions to expect.)